egs2/jsut/asr1/conf/tuning/train_transformer.yaml

```diff
@@ -0,0 +1,44 @@
+# network architecture
+# encoder related
+encoder: transformer
+encoder_conf:
+    input_layer: conv2d
+    num_blocks: 12
+    linear_units: 2048
+    dropout_rate: 0.1
+    output_size: 256
+    attention_heads: 4
+    attention_dropout_rate: 0.0
+
+# decoder related
+decoder: transformer
+decoder_conf:
+    input_layer: embed
+    num_blocks: 6
+    linear_units: 2048
+    drop_rate: 0.1
+
+# hybrid CTC/attention
+e2e_conf:
+    ctc_weight: 0.3
+    lsm_weight: 0.1
+    length_normalized_loss: false
+
+# optimization related
+optim: adam
+init: chainer
+accum_grad: 2
+grad_clip: 5
+patience: 0
+max_epoch: 50
+optim_conf:
+    lr: 5.0
+scheduler: noamlr
+scheduler_conf:
+    warmup_steps: 18000
+
+# minibatch related
+batch_size: 32
+max_length:
+    - 512
```

Comment:
`max_length` shouldn't be specified in config file because the appropriate value quite differs between feats_type=raw and feats_type=fbank_pitch.